shared/src/components/CodeExcerpt.test.tsx

```diff
@@ -104,4 +107,24 @@ describe('CodeExcerpt', () => {
             expect(span.textContent === 'test')
         }
     })
+
+    it('does not disable the highlighting timeout', () => {
+        /*
+            Because disabling the timeout should only ever be done in response
+            to the user asking us to do so, something that we do not do for
+            code excerpts because falling back to plaintext rendering is most
+            ideal.
+        */
+        const fetchHighlightedFileLines = sinon.fake.returns(of(HIGHLIGHTED_FILE_LINES))
+        const highlightRange = [{ line: 12, character: 7, highlightLength: 4 }]
+        render(
+            <CodeExcerpt
+                {...defaultProps}
+                highlightRanges={highlightRange}
+                fetchHighlightedFileLines={fetchHighlightedFileLines}
+            />
+        )
+        expect(fetchHighlightedFileLines.calledOnce).toBe(true)
```

Comment:
```suggestion
        sinon.assert.calledOnce(fetchHighlightedFileLines)
```

This gives a better assertion message:
"Expected function fetchHighlightedFileLines to be called once, but was called thrice"
vs
"Expected false to be true"